src/js/components/conversation.jsx

```diff
@@ -218,6 +218,7 @@ export class ConversationComponent extends Component {
                                      accentColor={ accentColor }
                                      linkColor={ linkColor }
                                      onLoad={ this.scrollToBottom }
+                                     entity={ message }
                                      {...message}
```

Comment:
if you are going to pass the whole message as a prop, remove the spreading here and use props from the entity directly instead, it will be much cleaner instead of having the same data at two places.